tools/getkeys/index.js

```diff
@@ -46,24 +103,6 @@ async function execAsync(command) {
     });
 }
 
-async function setEnv(name, value) {
-    const shell = process.env.SHELL ? process.env.SHELL.split('/').pop() : null;
-    const termProgram = process.env.TERM_PROGRAM;
-    const setString = `export ${name}="${value}"`;
-    switch (shell) {
-        case "bash":
-            const destFile = termProgram === "Apple_Terminal" ? "~/.bash_profile" : "~/.bashrc";
-            return execAsync(`${setString} && echo '${setString}' >> ${destFile}`);
```

Comment:
This code was previously executing the 'export' statement and using 'echo' to append the export to the rcFile.

I'm not sure if the purpose of the former was to "test" the export by running it, or if the author thought this might update the user's environment without requiring a restart (which it won't, because it executes in a spawned '/bin/sh'.)

I've switched these to use 'fs.appendFile()' instead.  The motivation was that I initially thought that avoiding 'echo' might fix the escaping issue, but that was not the case.  I left 'appendFile' because it just felt a cleaner (and makes it easier to do a single bulk write to the rcFile.)